inc/namespace.php

```diff
@@ -1119,7 +1122,7 @@ function get_logger( string $log_group, string $log_stream ) : LoggerInterface {
 			Altis\get_environment_name() . '/' . $log_group,
 			$log_stream,
 			null, // log retention when creating a new group (we disable this).
-			1000, // how many logs to send in a single batch.
+			$batch_size, // how many logs to send in a single batch.
```

Comment:
If there's a backlog of (eg) 20 logs and then this is set to 1, wouldn't that instruct it to send 20 requests to flush the existing ones? Feels to me like it would be better to force-flush when needed.

Reply:
@rmccue there couldn't be a backlog of 20 logs if this is set to 1, batch === buffer size

Reply:
I mean if something had previously called `get_logger()` for the same group - or do we not dedupe?

Reply:
No, there's no depublication